usagov/usagov-2021
# USAGOV-2071-Moderated-Content-View

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2071

## Description

Added the config files so that the moderated state is viewable in /admin/content/moderated

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Import config
- At localhost/admin/content/moderated you should now see a view of content in the "Draft" or "Needs Review" state. Previously this was empty (see Jira for photo if curious) 
- 

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
